### ISSUE TITLE
inmem datastet implementation

### ISSUE THREAD

**Issue by damooo** [pchampin/sophia_rs#137]
In 0.8 inmem implementation of datasets like `FastDataset` etc. are missing, while present in 0.7. Current inmem crate only contains implementations for inmem graphs.

**Comment by pchampin** [comment]
yes, 0.8 is still incomplete (hence the 'alpha' tag in the version). But inmem Dataset types will be back when 0.8 is released (soonish, hopefully).

Note that, in the meantime, you can still "default" implementations of Dataset (for example, any `Vec` or `HashSet` of `Quad`s implements `Dataset`: https://docs.rs/sophia/latest/sophia/dataset/trait.Dataset.html#implementors)
